mm/map/vm_region.c

```diff
@@ -125,7 +125,7 @@ FAR void *vm_map_region(uintptr_t paddr, size_t size)
         }
     }
 
-  return vaddr + (MM_PGMASK & paddr);
+  return (void *)((uintptr_t)vaddr + (MM_PGMASK & paddr));
 
 error:
   if (i)   /* undo alway mapped pages */
```

Comment:
Please fix the original issue that was in the file moving the comment to inside the: if (i) { /* here */ ... }

Reply:
I did that